### PR TITLE
Mig document page quality manual to v. 10.0

### PR DESCRIPTION
**Pull request by eugen-don**
Mig document page quality manual to v. 10.0

depends: 

- [x] https://github.com/OCA/management-system/pull/167

### CODE REVIEW

**Comment by max3903** [comment]
@eugen-don This branch needs to be rebased as well.

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
Ping @grindtildeath, @Learts

**Review by LeartS — APPROVED** [review]

File: document_page_quality_manual/README.rst [hunk 1]
```diff
@@ -54,6 +54,7 @@ Contributors
 * Odoo SA <info@odoo.com>
```
Comment (grindtildeath) [inline document_page_quality_manual/README.rst:54]: Can't select the exact lines but there are 9.0 to change to 10.0 on lines 31 and 45

**Review by grindtildeath — COMMENTED** [review]
Just links to fix, otherwise looks good. Thanks for your work !

File: document_page_quality_manual/README.rst (hunk 1 above)
Comment (eugen-don) [reply document_page_quality_manual/README.rst:54]: thanks

**Review by grindtildeath — APPROVED** [review]

**Review by ThomasBinsfeld — APPROVED** [review]

**Pull request merged**